mpas_analysis/test/test_mpas_xarray.py

```diff
@@ -2,73 +2,64 @@
 Unit test infrastructure for mpas_xarray.
 
 Xylar Asay-Davis, Phillip J. Wolfram
-12/07/2016
+02/15/2017
 """
 
 import pytest
 from mpas_analysis.test import TestCase, loaddatadir
 from mpas_analysis.shared.mpas_xarray import mpas_xarray
-import xarray as xr
 import pandas as pd
 
 
 @pytest.mark.usefixtures("loaddatadir")
-class TestNamelist(TestCase):
+class TestMpasXarray(TestCase):
 
     def test_subset_variables(self):
-        fileName = str(self.datadir.join('example_jan.nc'))
+        file_name = str(self.datadir.join('example_jan.nc'))
         timestr = ['xtime_start', 'xtime_end']
-        varList = ['time_avg_avgValueWithinOceanRegion_avgSurfaceTemperature']
+        var_list = ['time_avg_avgValueWithinOceanRegion_avgSurfaceTemperature']
```

Comment:
Another underscored variable-- I'm not going to note any more for the rest of this review....